android/src/main/java/nl/xguard/flic2/util/SisConsumer.java

```diff
@@ -0,0 +1,12 @@
+package nl.xguard.flic2.util;
```

Comment:
zijn deze bestanden nodig, zo ja, kunnen ze gehernoemd worden naar een verwijzing zonder Sis in de bestandsnaam?

Reply:
SisConsumer heb ik hernoemd naar Consumer. 

Deze class is vanaf api 24 direct beschikbaar, momenteel is de minimale api 23
https://developer.android.com/reference/java/util/function/Consumer

Reply:
👍